cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -14,17 +14,57 @@
  * limitations under the License.
  */
 
+#include "cudf/utilities/error.hpp"
```

Comment:
This should be decorated with `<>` and placed between `#include <cudf/utilities/bit.hpp>` and `#include <cudf/utilities/span.hpp>`

Reply:
Thanks for catching this 👍  Done.